R/prepareAnnotations.R

```diff
@@ -1,7 +1,13 @@
-#' Function to prepare tables and genomic ranges for
-#' transript reconstruction using a txdb object
-#' @title prepare annotations from txdb object or gtf file
-#' @param x A \code{TxDb} object or a gtf file
+#' This function takes a transcript annotation (path to gtf file/txdb
+#' object) and extends the metadata for each transcript.
+#' @title prepare annotations from path to gtf file or txdb object
+#' @param x A path to gtf file or a \code{TxDb} object
+#' @details For each transcript, its exons are ranked in the\code{exon_rank} column
+#' according to the strandedness and the transcript coordinates of the transcript. 
+#' The metadata also tells which gene each transcript (tx) belongs to 
+#' and its transcript equivalence class. A transcript equivalence class of a tx is
+#' the collection of transcripts where their exon junctions contain, in a continuous way, 
+#' the exon junctions of the tx. 
 #' @return A \code{GRangesList} object
 #' @importFrom methods is
 #' @importFrom GenomicFeatures exonsBy
```

Comment:
these 2 lines could be removed here

Reply:
as mentioned by @cying111 , we keep them.